orttraining/orttraining/core/session/training_session.cc

```diff
@@ -215,6 +215,12 @@ Status TrainingSession::ConfigureForTraining(
 
   ORT_RETURN_IF_ERROR(ApplyTransformationsToMainGraph(trainable_initializers, config.enable_gelu_approximation));
 
+  // std::string filename = "/bert_ort/liqun/test_out/mc_post_apply_trx.onnx";
+  // Save(filename, SaveOption::NO_RELOAD);
+
+  common::Status status = Model::Load("/bert_ort/liqun/test_out/mc_post_apply_trx_post_shape_infer.onnx", model_,
```

Comment:
>liqun [](start = 49, length = 5)

You'' remove/update this later, right?

Reply:
Yes.
This is the main thing to be solved if we want to the nuphar inference tool for training.